experimental/LieAlgebras/test/LieAlgebraModule-test.jl

```diff
@@ -1122,4 +1122,315 @@
+      end
+      R = root_system(:B, 3)
+      W = weyl_group(R)
+      
+      #x = epsilon
+      dict = Dict([1, 0, 0] => 1)
+      @test demazure_character(fundamental_weight(R, 1), one(W)) == dict
+      dict = Dict([0, 1, 0] => 1)
+      @test demazure_character(fundamental_weight(R, 2), one(W)) == dict
+      dict = Dict([0, 0, 1] => 1)
+      @test demazure_character(fundamental_weight(R, 3), one(W)) == dict
+      dict = Dict([1, 1, 1] => 1)
+      @test demazure_character(weyl_vector(R), one(W)) == dict
+
+      #x = s[1]
+      dict = Dict([-1, 1, 0] => 1,
+                  [1, 0, 0] => 1)
+      @test demazure_character(fundamental_weight(R, 1), W([1])) == dict
+      dict = Dict([0, 1, 0] => 1)
+      @test demazure_character(fundamental_weight(R, 2), W([1])) == dict
+      dict = Dict([0, 0, 1] => 1)
+      @test demazure_character(fundamental_weight(R, 3), W([1])) == dict
+      dict = Dict([-1, 2, 1] => 1,
+                  [1, 1, 1] => 1)
+      @test demazure_character(weyl_vector(R), W([1])) == dict
+      
+      #x = s[2]
+      dict = Dict([1, 0, 0] => 1)
+      @test demazure_character(fundamental_weight(R, 1), W([2])) == dict
+      dict = Dict([1, -1, 2] => 1,
+                  [0, 1, 0] => 1)
+      @test demazure_character(fundamental_weight(R, 2), W([2])) == dict
+      dict = Dict([0, 0, 1] => 1)
+      @test demazure_character(fundamental_weight(R, 3), W([2])) == dict
+      dict = Dict([2, -1, 3] => 1,
+                  [1, 1, 1] => 1)
+      @test demazure_character(weyl_vector(R), W([2])) == dict
+
+      #x = s[3]
+      dict = Dict([1, 0, 0] => 1)
+      @test demazure_character(fundamental_weight(R, 1), W([3])) == dict
+      dict = Dict([0, 1, 0] => 1)
+      @test demazure_character(fundamental_weight(R, 2), W([3])) == dict
+      dict = Dict([0, 1, -1] => 1,
+                  [0, 0, 1] => 1)
+      @test demazure_character(fundamental_weight(R, 3), W([3])) == dict
+      dict = Dict([1, 2, -1] => 1,
+                  [1, 1, 1] => 1)
+      @test demazure_character(weyl_vector(R), W([3])) == dict
+
+      #x = s[1]*s[3]
+      dict = Dict([-1, 1, 0] => 1,
+                  [1, 0, 0] => 1)
+      @test demazure_character(fundamental_weight(R, 1), W([1,3])) == dict
+      dict = Dict([0, 1, 0] => 1)
+      @test demazure_character(fundamental_weight(R, 2), W([1,3])) == dict
+      dict = Dict([0, 1, -1] => 1,
+                  [0, 0, 1] => 1)
+      @test demazure_character(fundamental_weight(R, 3), W([1,3])) == dict
+      dict = Dict([1, 1, 1] => 1,
+                  [-1, 2, 1] => 1,
+                  [-1, 3, -1] => 1,
+                  [1, 2, -1] => 1)
+      @test demazure_character(weyl_vector(R), W([1,3])) == dict
+
+      #x=s[3]*s[2]*s[1]
+      dict = Dict([1, 1, 1] => 1,
+                  [-1, 2, 1] => 1,
+                  [-1, 3, -1] => 1,
+                  [1, 2, -1] => 1)
+      @test demazure_character(fundamental_weight(R, 1), W([3,2,1])) == dict
```

Comment:
For this case magma tells me 
```julia
Highest weight decomposition of representation of:
    R: Simply connected root datum of dimension 3 of type B3
    Weights:
        [
            (0 0 0),
            ( 0  1 -2),
            (-1  1  0),
            (1 0 0),
            ( 0 -1  2)
        ]
    Multiplicities:
        [ 1, 1, 1, 1, 1 ]
```
so this seems to be a copy-paste issue. or rather this is the `dict` you define directly underneath this test.